CMakeFiles/cmake.check_cache

```diff
@@ -0,0 +1 @@
+# This file is generated by cmake for dependency checking of the CMakeCache.txt file
```

Comment:
This should not be part of the commit

Reply:
> `Replay/ReplayTests.RunReplay/issue8203` test is failing

Just removed all the whiteSpaces, any idea on how I'd fix this?